packages/config/src/use-env.ts

```diff
@@ -1,10 +1,21 @@
+import { deepSearch } from './utils';
+
 const envVarRegExp = /^@@/;
 
-export default function useEnv(config: { [key: string]: any }) {
+const isArrayProperty = (key: string, schema: { [key: string]: any }): boolean => {
```

Comment:
It should be _isStringArrayProperty_ then, if we are checking for 'array' and 'string' types.